docs/RBC_EN.md

```diff
@@ -0,0 +1,52 @@
+# RBC : Reliable Broadcast
+
+
+
+### overview
+
+The typical Broadcast, which depends on implementation, does not guarantee message delivery and does not consider situations in which the sender is crash (the situation in which the message cannot be sent to all). However, the RBC guarantees message delivery and partially solves the situation where the sender crashes. And the Atomic Broadcast introduced in this [paper](<https://eprint.iacr.org/2016/199.pdf>) is a Broadcast that guarantees the situation of the sender crash and guarantees the order of messages, and the RBC to be introduced is Atomic Broadcast.
+
+
+
+[The paper](<https://eprint.iacr.org/2016/199.pdf>) shows ABC (Atomic Broadcast) as ACS (Asynchronous Common Subset) using `threshold encryption`and ACS consists of RBC and BBA (Binary Byzantine Agreement) components. Among them, RBC performs reliable broadcasting and BBA is responsible for ordering of consensus and atomic messages.
+
+
+
+HBBFT does not send the entire data but sends the pieces by dividing the data by the number of nodes in the network. For this purpose, RBC uses a technique called `erasure coding`.
+
+### Erasure coding   
+
+Erasure coding is one of FEC (Forward Error Correction) methods and it can perform error detection and error correction. In RBC, Reed-solomon technique is used among erasure coding techniques. It splits the original data into n pieces and adds k parity. Then, even if a maximum of k data is lost, original data can be recovered if only n data are survived. Good Examples and detailed explanations can be found [here](<https://www.backblaze.com/blog/reed-solomon/>).
+
+<p align="center">
+        <img src="../img/rbc_algorithm.png" width="450px" height="600px"></img>
+</p>
+
+
+Assume that the content to be agreed on the network is Block.
+
+- There are N nodes on the network.
+
+- Split a block into N pieces. And erasure coding to generate N-2f parity.
+- S(j) is a fragment and the message to be delivered consists of VAL (h, b(j), s(j)). h: Merkle tree root of S(j), b(j): Merkle tree branch of S(j)
+
+
+
+1. When receiving VAL (h, b(j), s(j)) to P(sender), it multicasts ECHO (h, b(j), s(j)) to the node except the sender and itself.
```

Comment:
In `When receiving VAL (h, b(j), s(j)) to P(sender)`, `to` should changed to `from`.

Reply:
+ where is the description about
> except the sender and itself. 

?

Reply:
and `VAL(h, b(j), s(j))` should be fixed to `VAL(h, b(i), s(i))`

Reply:
I think it would be better there's more description about `sender`

Reply:
@zeroFruit sender and itself no need to receive messgae again. so paper descript as `multicast`. you think it needs additional description?